packages/mds-ingest-service/repository/entities/event-annotation-entity.ts

```diff
@@ -47,6 +48,13 @@ export class EventAnnotationEntity extends IdentityColumn(RecordedColumn(class {
 
   @Column('bigint', { transformer: BigintTransformer })
   latency_ms: Timestamp
+
+  @OneToOne(type => EventEntity, event => event.annotation, { nullable: true })
```

Comment:
I think the `nullable` is flipped on these new properties. An EventAnnotation will always have an Event but an Event may not always have an EventAnnotation.

Reply:
Looking into this.

Reply:
Made event and events_row_id non-nullable.